easybuild/easyconfigs/p/Pillow/Pillow-3.2.0-intel-2016a-Python-2.7.11-freetype-2.6.3.eb

```diff
@@ -0,0 +1,28 @@
+easyblock = 'PythonPackage'
+
+name = 'Pillow'
+version = '3.2.0'
+versionsuffix = '-Python-%(pyver)s'
+
+homepage = 'http://pillow.readthedocs.org/'
+description = """Pillow is the 'friendly PIL fork' by Alex Clark and Contributors.
+ PIL is the Python Imaging Library by Fredrik Lundh and Contributors."""
+
+toolchain = {'name': 'intel', 'version': '2016a'}
+
+source_urls = [PYPI_SOURCE]
+sources = [SOURCE_TAR_GZ]
+
+freetype_ver = '2.6.3'
+versionsuffix += '-freetype-%s' % freetype_ver
+dependencies = [
+    ('Python', '2.7.11'),
+    ('libjpeg-turbo', '1.4.2'),
+    ('zlib', '1.2.8'),
+    ('LibTIFF', '4.0.6'),
+    ('freetype', freetype_ver),
```

Comment:
then add it here too. Don't depend on indirect deps (else you can drop zlib too)